src/common/tiles/PersonTile/styled.js

```diff
@@ -9,71 +9,66 @@ export const Wrapper = styled.div`
   margin-top: 24px;
   justify-content: center;
   background-color: ${({ theme }) => theme.colors.lightGrey};
-`;
+`; */
 
 export const Container = styled.div`
-  width: 208px;
-  height: 339px;
+  padding: 16px;
+  word-break: break-word;
+  width: 100%;
+  height: 100%;
   background-color: ${({ theme }) => theme.colors.white};
   border-radius: ${({ theme }) => theme.radius.small};
-  padding: 16px;
+  box-shadow: ${({ theme }) => theme.shadow.mainShadow};
   display: flex;
   flex-direction: column;
-  box-shadow: ${({ theme }) => theme.shadow.mainShadow};
-
-  @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}) {
-    width: 136px;
-    height: 245px;
+  
+  @media (max-width:${({ theme }) => theme.breakpoints.large}) {
     padding: 8px;
   }
 `;
 
 export const ProfileImage = styled.img`
-  width: 177px;
-  height: 232.31px;
-  display: block;
-  flex-shrink: 0;
+  width: 100%;
   border-radius: ${({ theme }) => theme.radius.small};
-  object-position: top;
-  object-fit: cover;
-
-  @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}) {
-    width: 120px;
-    height: 178px;
-  }
+  aspect-ratio: 2/3;
```

Comment:
nieznana właściwość dla mnie :D

Reply:
Służy do skalowania i ustawiania jednolitej proporcji elementu. To moje niedawne odkrycie, bo miałem problem z równym dostosowaniem szarego obrazka noPicture ze zdjęciami z API. A ustawienie ich na sztywno było trochę problematyczne.